src/BloomExe/Book/Book.cs

```diff
@@ -388,6 +411,37 @@ public HtmlDom GetPreviewXmlDocumentForPage(IPage page)
 			return pageDom;
 		}
 
+		// Differs from GetPreviewXmlDocumentForPage() by not adding the three stylesheets
+		// adding them will full paths seems to be diastrous. I think cross-domain rules
+		// prevent them from being loaded, and so we lose the page size information, and the
+		// thumbs come out random sizes. Not sure why this isn't a problem in GetPreviewXmlDocumentForPage.
```

Comment:
Should I do the investigation to figure out whether any of the callers of GetPreviewXmlDocumentForPage actually need those extra stylesheets? At least some of them seemed to be already present when I tried to use it, but I don't know whether that is true for other clients. If we could get rid of that difference we could get back to a single method here (with the client calling AddFillerTextToEmptyEditDivs when appropriate).